userappwithbroker/build.gradle

```diff
@@ -70,7 +70,7 @@ dependencies {
     implementation "com.google.android.material:material:$rootProject.ext.materialVersion"
 
     localImplementation(project(':adal'))
-    distImplementation("com.microsoft.aad:adal:1.16.3") {
+    distImplementation("com.microsoft.aad:adal:1.16.3-hf2") {
```

Comment:
Should we use the latest here too?

Reply:
guess so, but let's defer it for now, we can make a change to dev, since this is a merge PR.